src/libs/logos.js

```diff
@@ -13,6 +13,8 @@ const brandLogoMaker = (size, color = false) =>
     style: { height: size, marginRight: '1.5rem' },
   });
 
+// Needs to be capitalized to be a TSX Component
+export const RegistrationLogo = () => registrationLogo();
```

Comment:
For what it's worth, this could be simplified a little to avoid the extra function call.

```suggestion
export const RegistrationLogo = registrationLogo;
```

Or even better... Register is the only place this function is used, so let's just rename it instead of creating an alias.